packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -713,7 +714,13 @@ const QueryResultCellRenderer = observer(
           onMouseUp={(event) => mouseUp(event)}
           onMouseOver={(event) => mouseOver(event)}
         >
-          <span>{cellValue}</span>
+          {isValidURL(cellValue) ? (
+            <a href={cellValue} target="_blank" rel="noreferrer">
+              {cellValue}
+            </a>
+          ) : (
+            <span> {cellValue} </span>
```

Comment:
why adding the spaces around cell value?
```suggestion
            <span>{cellValue}</span>
```